config/armada/config.yaml

```diff
@@ -2,6 +2,7 @@
 grpcPort: 50051
 httpPort: 8080
 metricsPort: 9000
+corsAllowedOrigins: ["http://localhost:3000"]
```

Comment:
I would prefer default to be empty and set inside helm chart to lookout domain. It is minor but we don't want any localhost to talk to the api.